### PR TITLE
Fixing check - EECORE-1266

### PR DESCRIPTION
**Pull request by dougblackjr**
## Overview

Fixes ReCAPTCHA check

## Nature of This Change

- [X] 🐛 Fixes a bug
- [ ] 🚀 Implements a new feature
- [ ] 🛁 Refactors existing code
- [ ] 💅 Fixes coding style
- [ ] ✅ Adds tests
- [ ] 👽 Adds new dependency
- [ ] 🔥 Removes unused files / code
- [ ] 🔒 Improves security 

## Is this backwards compatible?

- [ ] Yes
- [X] No

### CODE REVIEW

**Review by ops-andy — DISMISSED** [review]

File: system/ee/ExpressionEngine/Library/Captcha.php [hunk 1]
```diff
@@ -48,7 +48,7 @@ public function create($old_word = '', $force_word = false)
             $score_threshold = ee()->config->item('recaptcha_score_threshhold');
             $action_id = ee()->functions->fetch_action_id('member', 'recaptcha_check');
 
-            $js = "<script> var csrf_token='".CSRF_TOKEN."'; var endpoint ='".ee()->functions->fetch_site_index(0, 0) . QUERY_MARKER . 'ACT=' . $action_id."'; var key ='".$key."'; </script>";
+            // $js = "<script> var csrf_token='".CSRF_TOKEN."'; var endpoint ='".ee()->functions->fetch_site_index(0, 0) . QUERY_MARKER . 'ACT=' . $action_id."'; var key ='".$key."'; </script>";
```
Comment (intoeetive) [inline system/ee/ExpressionEngine/Library/Captcha.php]: It looks like commenting this out will make `endpoint` in JS and `$js` in PHP undefined

File: system/ee/installer/updates/ud_6_00_04.php [hunk 2]
```diff
@@ -48,7 +48,7 @@ private function addreCaptchaAction()
         ee()->db->insert('actions', array(
             'class' => 'Member',
             'method' => 'recaptcha_check',
-            'csrf_exempt' => 0
+            'csrf_exempt' => 1
```
Comment (intoeetive) [inline system/ee/installer/updates/ud_6_00_04.php]: If we're making this `csrf_exempt` we also need to make sure it's still excempt when updating from `6.0.6` to `6.1.0`, so need to perform change in latest `ud_` file as well

**Review by intoeetive — CHANGES_REQUESTED** [review]

File: system/ee/ExpressionEngine/Library/Captcha.php (hunk 1 above)
Comment (dougblackjr) [reply system/ee/ExpressionEngine/Library/Captcha.php]: @intoeetive Good catch. Updated on latest commits

File: system/ee/installer/updates/ud_6_00_04.php (hunk 2 above)
Comment (dougblackjr) [reply system/ee/installer/updates/ud_6_00_04.php]: @intoeetive Per conversation, moving this to 6.1 check, and reverting this change.

File: system/ee/ExpressionEngine/Library/Captcha.php [hunk 3]
```diff
@@ -48,7 +48,7 @@ public function create($old_word = '', $force_word = false)
             $score_threshold = ee()->config->item('recaptcha_score_threshhold');
             $action_id = ee()->functions->fetch_action_id('member', 'recaptcha_check');
 
-            $js = "<script> var csrf_token='".CSRF_TOKEN."'; var endpoint ='".ee()->functions->fetch_site_index(0, 0) . QUERY_MARKER . 'ACT=' . $action_id."'; var key ='".$key."'; </script>";
+            $js = "var endpoint ='".ee()->functions->fetch_site_index(0, 0) . QUERY_MARKER . 'ACT=' . $action_id."'; var key ='".$key."'; </script>";
```
Comment (intoeetive) [inline system/ee/ExpressionEngine/Library/Captcha.php]: Thinking of these further, should we use something less generic as variable names here? Or add namespace

**Review by intoeetive — CHANGES_REQUESTED** [review]

File: system/ee/ExpressionEngine/Library/Captcha.php (hunk 3 above)
Comment (dougblackjr) [reply system/ee/ExpressionEngine/Library/Captcha.php]: @intoeetive Fixed on latest commit.

File: system/ee/ExpressionEngine/Library/Captcha.php [hunk 4]
```diff
@@ -48,7 +48,7 @@ public function create($old_word = '', $force_word = false)
             $score_threshold = ee()->config->item('recaptcha_score_threshhold');
             $action_id = ee()->functions->fetch_action_id('member', 'recaptcha_check');
 
-            $js = "<script> var csrf_token='".CSRF_TOKEN."'; var endpoint ='".ee()->functions->fetch_site_index(0, 0) . QUERY_MARKER . 'ACT=' . $action_id."'; var key ='".$key."'; </script>";
+            $js = "var eeRecaptchaEndpoint ='".ee()->functions->fetch_site_index(0, 0) . QUERY_MARKER . 'ACT=' . $action_id."'; var key ='".$key."'; </script>";
```
Comment (intoeetive) [inline system/ee/ExpressionEngine/Library/Captcha.php]: the `key` variable is still not unique

**Review by intoeetive — CHANGES_REQUESTED** [review]

File: system/ee/ExpressionEngine/Library/Captcha.php (hunk 4 above)
Comment (dougblackjr) [reply system/ee/ExpressionEngine/Library/Captcha.php]: Fixed on latest commit. EDIT: Commit is incoming, on slow public wifi.

**Review by intoeetive — APPROVED** [review]

**Pull request merged**